cogs/Helpful.py

```diff
@@ -0,0 +1,105 @@
+import discord
+from discord.ext import commands
+import os
+import random
+
+
+class Helpful(commands.Cog):
+    
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print('Helpful Cog is online')
+
+    @commands.command()
+    async def report7z(self, ctx):
+        """Report7z reminder"""
+        await ctx.send("Please send report7z.")
+        await ctx.send("https://dw.ngmansion.xyz/doku.php?id=en:guide:febuildergba:report7z")
+
+    @commands.command()
+    async def howtomod(self, ctx):
+        """Gives information on how to use the !mod command."""
+        await ctx.send("First, have Developer Mode enabled (Settings -> Appearance -> Developer Mode).")
+        await ctx.send("Then, click the `...` by the offending message, and click \"Copy Link\".")
+        await ctx.send("Then simple say !mod <n> <link>, where <n> is the rule it violates, and <link> is the pasted link to the message.")
+        await ctx.send("If you do not have Developer Mode, you may instead of a link, write a short description of where the infraction took place, and by who.")
+        await ctx.send("Note that after requesting moderation, the message requesting moderation will be removed.")
+
+
+    @commands.command()
+    async def goldmine(self, ctx):
+        """everything you ever wanted"""
+        embed=discord.Embed(title="Unified FE Hacking Dropbox", url='https://www.dropbox.com/sh/xl73trcck2la799/AAAMdpNSGQWEzYkLEQEiEhGFa?dl=0', description="All the hacking resources you could ever need, in one place", color=0xefba01)
+        # embed.set_thumbnail(url='http://i.imgur.com/Bg5NSga.png')

```

Comment:
Is this line actually doing anything and commented for convenience, or can we just remove it?

Reply:
removed this line